Automattic/wp-calypso
# Store Orders: Update the refund status to "payment status", show awaiting payment message

Fixes #17760 by updating the message show to the user to "Awaiting payment", along with a "Mark as Paid" button.

<img width="739" alt="screen shot 2017-09-05 at 11 08 11 am" src="https://user-images.githubusercontent.com/541093/30068440-d33b8fc4-922b-11e7-8071-a423d2edc8d5.png">

This PR also renames the `OrderRefundCard` component & folder to `OrderPaymentCard`, since it displays payment status, not just refund status. The rename affects some function names, and the class names in the component, but these shouldn't change any other functionality.

**To test**

- View an order that is pending payment or on hold (either by placing an order using check payment or bank transfer payment, or by changing the status of an existing order)
- The payment status line should now say "Awaiting payment…"
- Click the "Mark as Paid" button
- The order should immediately update to "processing", and the payment message should update.

Checking that it still displays correctly for other order statuses would also be helpful 🙂
